main.cpp

```diff
@@ -83,70 +101,112 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+void payRent(int amount)
+{
+    ignoreUnused(amount);
+}
 /*
  2)
  */
-
+void makeCoffee(char size = 'm',int strength = 2)
+{
+    ignoreUnused(size,strength);
+}
 /*
  3)
  */
-
+bool canIAfford(int cost, float tax, float bankBalance =100.98f)
+{
+    ignoreUnused(cost,tax,bankBalance);
+    return {};
+}
 /*
  4)
  */
-
+double findCoordinates(int postcode, int houseNumber=289)
+{
+    ignoreUnused(postcode,houseNumber);
+    return {};
+}
 /*
  5)
  */
+void tieLaces (int shoe=1, int hand=0)
+{
+    ignoreUnused(shoe,hand);
```

Comment:
- [ ] add spaces between your parameters, default values and things you pass to functions so they are easier to read